### ISSUE TITLE
Talkback Accessibility - Please have roles types describe that buttons are buttons. 

### ISSUE THREAD

**Issue by caltseng** [uservoice/uservoice-android-sdk#249]
User Experience:
Users who rely on screen readers will get confused about the element if it does not read the role of an element.
 
Steps to reproduce:
Step 1: Switch on Talkback.
Step 2: Open Staffhub app in android phone.
Step 3: Navigate to ‘Open Navigation drawer’ button using tab key and select it using enter key.
Step 4: Navigate to ‘Help & Feedback’ in list item and select it.
Step 5: Navigate to ‘Contact support’ button using Shift + alt + left/right arrow keys and select it.
Step 6: Using Shift + alt + left/right arrow keys traverse through the elements.
Step 7: Observe the Talkback speech.
 
Actual result:
Role is not defined for ‘Attach Screenshot’ and ’Contact us’ buttons. Talkback just reads as “Attach screenshot”. Also, talkback reads 'Contact us' as "Contact US" (US like country name ).
 
Expected result:
Talkback should read 'Attach Screenshot' control as 'Attach Screenshot button' and ’Contact us’ control as 'Contact us button'.
Also, Talkback should read 'us' of 'Contact us' properly and not like country's name 'US'. 

Note:
This issue is also found in 'Help & Feedback' page > 'Suggest Idea' button > Talkback doesn't read the role of 'Search' & 'Submit Idea' button. Refer to screenshot: 'MAS33A_Suggest Idea_Role is missing '.
